atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/timestamp/InDbTimestampBoundStore.java

```diff
@@ -197,8 +198,20 @@ private void createLimit(Connection connection, long limit) throws SQLException
 
     private void createTimestampTable(Connection connection) throws SQLException {
         try (Statement statement = connection.createStatement()) {
-            statement.execute(String.format("CREATE TABLE IF NOT EXISTS %s ( last_allocated int8 NOT NULL )",
-                    timestampTable.getQualifiedName()));
+            if (getDbType(connection).equals(DBType.ORACLE)) {
+                try {
+                    statement.execute(String.format("CREATE TABLE %s ( last_allocated NUMBER(38) NOT NULL )",
+                            timestampTable.getQualifiedName()));
+                } catch (PalantirSqlException e) {
+                    if (!e.getMessage().contains(ORACLE_ALREADY_EXISTS_ERROR)) {
+                        log.error(e.getMessage(), e);
```

Comment:
This will look weird as the message will be output twice. The first argument should probably be something like `"An error occurred creating the timestamp table"`